include/memory_layout.h

```diff
@@ -20,15 +20,19 @@
 #define INITRD_START       KERNEL_PM_CODE_END
 #define MAX_INITRD_END     0x02A00000
 
+#define MAX_KERNEL_SIZE    (KERNEL_PM_CODE_END - KERNEL_PM_CODE)
+#define MAX_INITRD_SIZE    (MAX_INITRD_END - INITRD_START)
+
+/* parameters for ReactOS have to be here */
+#define FREELDR_LOAD_AREA  (u8 *)0x00001000
+#define FREELDR_MAX_SIZE   0x80000
+
 #define MEMORYMANAGERSTART MAX_INITRD_END
 #define MEMORYMANAGEREND   0x039FFFFF
+#define MEMORYMANAGERSIZE  (MEMORYMANAGEREND - MEMORYMANAGERSTART)
 
 #define STACK_TOP 0x03C00000
 
-#define MAX_KERNEL_SIZE    (KERNEL_PM_CODE_END - KERNEL_PM_CODE)
-#define MEMORYMANAGERSIZE  (MEMORYMANAGEREND - MEMORYMANAGERSTART)
-#define MAX_INITRD_SIZE    (MAX_INITRD_END - INITRD_START)
```

Comment:
Why was this moved?

Reply:
For better grouping of definitions.

Reply:
Ah, I can see what you mean now - because kernel and initrd only affect Linux